plugins/playstation/src/playstation-plugin.vala

```diff
@@ -0,0 +1,32 @@
+// This file is part of GNOME Games. License: GPLv3
+
+private class Games.PlayStation : Object, Plugin {
+	private const string FINGERPRINT_PREFIX = "playstation";
+	private const string MIME_TYPE = "application/x-cue";
```

Comment:
@hadess are 0xD368 and 0xD3A8 too far away in the file to search for the keyfile in shared-mime-info?